File: TenMinsJava/day54.java
// Check if string is 8 digits
package TenMinsJava;

import java.util.Scanner;

public class day54 {
    public static void main(String[] args) {
        Scanner scan =  new Scanner(System.in);
        System.err.println("Enter 8 digit number");
        String s = scan.nextLine();
        if (s.length() == 8) {
            if (s.matches("\\d+")) {
                System.err.println("nice");
            }
            else {
                System.err.println("not a letter bro");
            }
        }
        else {
            System.err.println("insufficient length");
        }
    }
}

File: TenMinsPy/day54.py
# Check if string is 8 digits

class day54:
    def __init__(self):
        digit = 0
        s = input('enter 8 digit number\n')
        for c in s:
            if c.isdigit():
                digit += 1
        if len(s) == 8 and digit == 8:
            print('nice')
        else:
            print("insufficient length or not a digit")

day54()